src/Resources/config/services.yml

```diff
@@ -101,12 +101,16 @@ services:
     contao.fragment.registry:
         class: Contao\CoreBundle\Fragment\FragmentRegistry
 
-    contao.fragment.renderer:
-        class: Contao\CoreBundle\Fragment\FragmentRenderer
+    contao.fragment.handler:
+        class: Contao\CoreBundle\Fragment\FragmentHandler
+        decorates: fragment.handler
         arguments:
+            - ""
```

Comment:
The first argument has to implement `ContainerInterface`.

Reply:
It is set by the compiler pass (see `FragmentRendererPass`)